gradle/dependencies.gradle

```diff
@@ -164,6 +165,7 @@ libs += [
   apachedsMavibotPartition: "org.apache.directory.server:apacheds-mavibot-partition:$versions.apacheds",
   apachedsJdbmPartition: "org.apache.directory.server:apacheds-jdbm-partition:$versions.apacheds",
   argparse4j: "net.sourceforge.argparse4j:argparse4j:$versions.argparse4j",
+  awaitility: "org.awaitility:awaitility:$versions.awaitility",
```

Comment:
we add new dependency to Kafka only if no other option is available. This is because they add the overhead of upgrading in case of vulnerability etc. 

Can you please help me understand why the existing code wouldn't work? Alternatively, can we add a utility (or reuse existing one) in TestUtils to implement the same behaviour?

Reply:
Yes this makes sense. I later found the similar code in TestUtils so discarded this PR. Thanks for the info still.